common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_wsc.h

```diff
@@ -0,0 +1,44 @@
+///////////////////////////////////////
+// AUTO GENERATED FILE - DO NOT EDIT //
+///////////////////////////////////////
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * Copyright (c) 2016-2019 Tomer Eliyahu (Intel Corporation)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#ifndef _TLVF_WSC_WSC_ATTRIBUTES_HELPERS_H_
+#define _TLVF_WSC_WSC_ATTRIBUTES_HELPERS_H_
+
+#include <beerocks/bcl/beerocks_utils.h>
+#include <tlvf/WSC/WSC_Attributes.h>
+
+namespace WSC {
+    enum class eBssType : uint8_t {
```

Comment:
Why enum class and not regular enum?

Reply:
no real reason - I will change to regular enum.

Reply:
Are you planning to use this enum without `WSC::eBssType::`?
If not - keep the enum class

Reply:
Yes - I will change it to use uint8_t so better to use simple enum as the values are only bitmasks.